charts/shenyu/values.yaml

```diff
@@ -37,6 +37,22 @@ dataSource:
     # mysql6-8 : com.mysql.cj.jdbc.Driver
     driverClass: com.mysql.cj.jdbc.Driver
     connectorVersion: 8.0.23
+  pg:
+    ip:
+    port: 5432
+    username: postgres
+    password:
+    # pg driver class name
+    driverClass: org.postgresql.Driver
+    connectorVersion: 42.5.0
+  oracle:
```

Comment:
I think oracle should add in another pr

Reply:
Maybe just let it be for now? I have the calibration in the subsequent steps.

<img width="966" alt="image" src="https://user-images.githubusercontent.com/36830265/193407590-8ae56a6d-5517-45bf-8039-2636945defb7.png">

How about just adding some comments like this?

<img width="478" alt="image" src="https://user-images.githubusercontent.com/36830265/193407618-8abe8339-a5cb-491c-b72d-f55ecc1b8b19.png">

Reply:
very good